readthedocs/rtd_tests/tests/test_subprojects.py

```diff
@@ -143,6 +143,20 @@ def test_excludes_existing_subprojects(self):
             [''],
         )
 
+    def test_exclude_self_project_as_subproject(self):
+        user = fixture.get(User)
+        project = fixture.get(Project, users=[user])
+        subproject = project
+
+        form = ProjectRelationshipForm(
+            {'child': subproject.pk},
+            project=project,
+            user=user
+        )
```

Comment:
I think `is_valid` should be called and tested that it return `False`

Reply:
@stsewd take a look at this comment.

If you don't calle `is_valid` all your validations (`clean_` methods) are not called.